### ISSUE TITLE
Schemes with multiple targets only build the last target

### ISSUE THREAD

**Issue by jspahrsummers** [Carthage/Carthage#128]
This is an issue if, e.g., a framework scheme also builds its own unit tests. In such a case, Carthage will pick up settings and build products from the test target, instead of the framework itself.

Originally from https://github.com/Carthage/Carthage/issues/120.

/cc @RuiAAPeres

**Comment by RyogaK** [comment]
Hi, @jspahrsummers
I'm using carthage 0.18. However, it seems this problem is here again.
I'm not familiar with carthage's code base, so I don't know what happens.
At least, in my forked project, a scheme won't picked up by carthage for now.
https://github.com/RyogaK/MRProgress/tree/carthage

Could you help me?

**Comment by ikesyo** [comment]
@RyogaK I'm not sure what the problem really it is, but I can't build the `MRProgress-AFNetworking` even in Xcode GUI. You should make it sure that you can build targets / schemes you want in Xcode, and please file a new issue for the specific case. This was closed too earlier days and should not be appropriate.

**Comment by RyogaK** [comment]
@ikesyo Thanks for your quick response.
Hmm, We can build all schemes in our all Xcode environment, so I wondered your xcode's problem.
Well, I investigate more and make issue again if needed.

Thanks!

**Comment by ikesyo** [comment]
@RyogaK FYI I could successfully build the project by Carthage with `carthage build --platform ios --no-skip-current` once after I ran `carthage checkout`. That built both `MRProgress` and `MRProgress-AFNetworking`.

**Comment by RyogaK** [comment]
@ikesyo Thanks for your report!
I saw sometime carthage said as builds are completed, but MRProgress.framework is not in `Carthage/Build/iOS`.
If I remove `MRProgress-AFNetworking` scheme, and I try to `carthage bootstrap` again, that will be made and there is here. 😅 

I saw following cases randomly, so I'm very confused.
Randomly means I just clear Xcode's cache and Carthage folder, and then I call `carthage bootstrap` again. I was repeated this.

- Case1: Carthage doesn't pick up `MRProgress` scheme.
- Case2: Carthage picks up that scheme, but the framework is not placed.

If I specify the library and call `carthage build` like `carthage build MRProgress`, the framework will appear. This problem will occur on Circle CI(OSX), so I totally couldn't understand what's the cause.
